### DISCUSSION TITLE
Integrate with Nuxt/Image

### DISCUSSION THREAD

**Discussion by HeinKoZin** [ivodolenc/nuxt-lazysizes#12] [Q&A]
How I can I use with Nuxt/image. I got large size of images from my api. I want to compress images to low file size. I currently use Nuxt/image to compress my images.

**Comment by ivodolenc** [comment]
Hi, thanks for your interest in this module. 

I'm not familiar with the `nuxt-image` module so I should first read its documentation to know the correct answer.

But it is very likely that the modules could work together. Do you have any example code that you have already tried?

**Reply by HeinKoZin** [reply]
I had tried nuxt-image with lazysizes.

`<nuxt-img data-src="https://placeholder.com/" class="lazyload" format="webp" />`

But nuxt-image required src attribute.
